app/src/main/java/co/nimblehq/compose/crypto/ui/components/chartintervals/ChartIntervalsButton.kt

```diff
@@ -38,13 +38,20 @@ fun ChartIntervalsButtonGroup(
                 Color.Transparent
             }
 
+            val textColor = if (selectedColor.value == index) {
+                Color.White
+            } else {
+                Color.OsloGray
+            }
+
             ChartIntervalsButton(
                 modifier = Modifier
                     .requiredWidth(Dp45)
                     .background(
                         color = backgroundColor,
                         shape = RoundedCornerShape(Dp12)
                     ),
+                textColor = textColor,
```

Comment:
Should we make use of `coinPriceIntervalTextColor`?

Reply:
@luongvo Ops. `coinPriceIntervalTextColor` is redundant. Let me remove it.